content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Gateway
```

Comment:
```suggestion
title: Gateway APIs
```
We could also call it “Gateway API”, which was the name for beta. It's tricky; neither is right.

- this is a family of APIs
- people have been calling that family of APIs “Gateway API” for a while, including since beta
- there's no style guide (that I know of) for the docs for extensions like Gateway API or Cluster API.

Reply:
Yes, naming for this project has been tricky. It started off plural with the name "Service APIs" but was changed to the current singular form, being named after the primary API kind. I dropped API from the title for consistency with the Ingress title but I'll add it as you suggest.